cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerServiceLauncher.java

```diff
@@ -269,6 +269,14 @@ public void run() {
             }
           }
 
+          if (cConf.getBoolean(Constants.TaskWorker.TASK_WORKER_LIVENESS_PROBE_ENABLED)) {
+            if (twillPreparer instanceof ExtendedTwillPreparer) {
+              twillPreparer = ((ExtendedTwillPreparer) twillPreparer)
+                  .setProbe(TaskWorkerTwillRunnable.class.getSimpleName(),
```

Comment:
nit: please rename to `setProbeConfig` instead of `setProbe` as this function sets the config.